src/components/DashboardLayout/InstanceNestedBarStates.tsx

```diff
@@ -0,0 +1,93 @@
+import { useState } from 'react';
+import { RadioGroup } from '@headlessui/react';
+import { InstanceContext } from 'data/InstanceContext';
+import { useContext, useEffect } from 'react';
+import useAnalyticsEventTracker from 'utils/hooks';
+import clsx from 'clsx';
+import { BrowserLocationContext } from 'data/BrowserLocationContext';
+import InstanceCard from 'components/InstanceCard';
+import InstanceTabListMap from '../../data/InstanceTabListMap';
+
+export const InstanceNestedBarStates = ({
+  className = '',
+  children,
+}: {
+  className?: string;
+  children?: React.ReactNode;
+}) => {
+  const {
+    instanceList: instances,
+    selectedInstance,
+    isReady,
+  } = useContext(InstanceContext);
+
+  const gaEventTracker = useAnalyticsEventTracker('Instance List');
+  const { setPathname } = useContext(BrowserLocationContext);
+  const [setActive, setActiveTab] = useState(location.pathname.split('/')[2]);
+
+  useEffect(() => {
+    if (!isReady) return;
+    gaEventTracker(
+      'View',
+      'Instance List',
+      true,
+      Object.keys(instances).length
+    );
+  }, [isReady, instances]);
+
+  useEffect(() => {
+    setActiveTab(location.pathname.split('/')[2]);
+  }, [location.pathname]);
+
+  const uuid = selectedInstance?.uuid;
+  if (!selectedInstance || !uuid) {
+    return <div></div>;
+  }
+
+  return (
+    <RadioGroup
+      className={`gap mx-1 flex min-h-0 flex-col gap-y-1 overflow-y-auto overflow-x-hidden px-1 child:w-full ${className}`}
+      value={selectedInstance}
+    >
+      <RadioGroup.Label className="text-small font-bold leading-snug text-gray-faded/30">
+        SELECTED INSTANCE
+      </RadioGroup.Label>
+
+      <InstanceCard {...selectedInstance} />
+
+      {selectedInstance &&
+        InstanceTabListMap[selectedInstance.game_type].map((tab) => (
```

Comment:
The last button here is cut off visually
<img width="120" alt="image" src="https://user-images.githubusercontent.com/44907675/218777287-241389c7-bdd2-435e-b6de-029c0ff71ad3.png">

Reply:
These buttons are also missing `focus-visible` states (keyboard tabbing)

Reply:
Buttons are still missing `focus-visible` state